src/pretalx/cfp/templates/cfp/event/submission_questions.html

```diff
@@ -4,12 +4,27 @@
 {% load bootstrap4 %}
 {% block title %}{{ request.event }}{% endblock %}
 {% block inner %}
-    <h2>{% trans "This looks good!" %}</h2>
+    <h2>{% trans "Tell us more!" %}</h2>
     <p>
         {% blocktrans trimmed with step=wizard.steps.step1 steps=wizard.steps.count %}
             Before we can save your submission, we have some more questions for you.
             But rest assured, you reached step {{ step }} of {{ steps }} already, we are nearly there!
         {% endblocktrans %}
     </p>
-    {% bootstrap_form wizard.form %}
+    {% if submission_question_names %}
+    <h4>{% trans "… about your submission:" %}</h4>
+        {% for field in wizard.form %}
+            {% if field.name in submission_question_names %}
```

Comment:
This doesn't seem very clean to me. The `group1`/`group2` methods variant showcased in [this SO answer](https://stackoverflow.com/a/10367761/2486196) seems a little better, without getting into the whole formset madness. Maybe you like it too?